apps/ledger-live-desktop/src/renderer/modals/EditTransaction/steps/GenericStepConnectDevice.jsx

```diff
@@ -0,0 +1,114 @@
+// @flow
```

Comment:
is there a way to use the same sendAmount as the send flow ?
I see no point on duplicating a generic file and it will hard to maintain

Reply:
fixed